python/interpret_community/mimic/mimic_explainer.py

```diff
@@ -288,7 +289,6 @@ def __init__(self, model, initialization_examples, explainable_model, explainabl
             # Index the categorical string columns for training data
             self._column_indexer = initialization_examples.string_index(columns=categorical_features)
             self._one_hot_encoder = None
```

Comment:
why was this removed?
```
             explainable_model_args[LightGBMParams.CATEGORICAL_FEATURE] = categorical_features 
```

Reply:
oh I guess you moved it to line 347, I guess that's ok, my only slight concern is now we are doing the same checks in multiple places:
```
is_tree_model = explainable_model.explainable_model_type == ExplainableModelType.TREE_EXPLAINABLE_MODEL_TYPE
        if is_tree_model and self._supports_categoricals(explainable_model):
```
but it's not expensive so I think it's ok